src/main/java/io/gravitee/policy/cache/invoker/CacheInvoker.java

```diff
@@ -153,7 +157,7 @@ private void storeInCache(String cacheId, HttpHeaders httpHeaders, int status, B
         Completable
             .fromAction(() -> {
                 final var resp = new CacheResponse();
-                resp.setContent(buffer);
+                resp.setContent(Buffer.buffer(Base64.getEncoder().encode(buffer.getBytes())));
```

Comment:
We could probably add a unit test that validates the behavior with binary versus text based content. WDYT?

Reply:
sure test added